ui/app/components/clients/current.js

```diff
@@ -15,10 +16,30 @@ export default class Current extends Component {
   @tracked selectedAuthMethod = null;
   @tracked authMethodOptions = [];
 
-  get latestUpgradeData() {
-    // e.g. {id: '1.9.0', previousVersion: null, timestampInstalled: '2021-11-03T10:23:16Z'}
-    // version id is 1.9.0 or earliest upgrade post 1.9.0, timestamp is RFC3339
-    return this.args.model.versionHistory[0] || null;
+  get upgradeVersionHistory() {
+    const versionHistory = this.args.model.versionHistory;
+    if (!versionHistory || versionHistory.length === 0) {
+      return null;
+    }
+
+    // get upgrade data for initial upgrade to 1.9 and/or 1.10
+    let relevantUpgrades = [];
+    const importantUpgrades = ['1.9', '1.10'];
+    importantUpgrades.forEach((version) => {
+      let findUpgrade = versionHistory.find((versionData) => versionData.id.match(version));
+      if (findUpgrade) relevantUpgrades.push(findUpgrade);
+    });
+
+    // if no history for 1.9 or 1.10, get first upgrade available
+    if (Object.entries(relevantUpgrades).length === 0) {
+      relevantUpgrades.push({
```

Comment:
If a customer skips the relevant upgrades, for example is on 1.8 and jumps to 1.11, then we need to capture whatever their first stored upgrade date is. We still want to show a warning to explain any potential data discrepancies: 

![image](https://user-images.githubusercontent.com/68122737/164787815-3b43558f-3a1a-4dbf-bc94-f70c7ef795da.png)

Reply:
But good point that we do need to account for if a vault customer _starts_ at a version >= 1.11 
Since `previousVersion = null` whether a user upgrades from pre-1.9 or their first version of vault is >= 1.11 we have no way of telling the difference